### PR TITLE
API Builder terminology update and section redraft

### PR DESCRIPTION
**Pull request by SueSmith**
Added API Builder terminology, also redrafted design and develop section (excluding stuff that had been updated recently).

### CODE REVIEW

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 1]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
-Displays the number of new APIs created in the team. However, note that new versions of an API created are not accounted for.  
+The average response time is calculated based on the time taken to return the response of the request and the number of requests sent on a particular day.
 
-### Active APIs
+[![team api req](https://assets.postman.com/postman-docs/Reports-AvgRespTime.png)](https://assets.postman.com/postman-docs/Reports-AvgRespTime.png)
 
-Displays the number of active APIs in the team. Postman considers an API active if at least one request was sent through a collection linked to an API.
+> In this example, the response time is shown for Jun 12. No response is received on other dates because no request is sent.
+>
+> Summary average response size may not match with the average of the graph, because raw data is accounted for when calculating averages. In order to determine an accurate average, Postman does not calculate averages using data from already derived averages.
```
Comment (arlemi) [inline src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: > In order to determine an accurate average, Postman does not calculate averages using data from already derived averages.

🤯 (just pointing out that my brain was not ready for that sentence, there's nothing wrong with it)

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 2]
```diff
@@ -1,141 +1,104 @@
...
+>
+> ![Share API in Dashboard](https://assets.postman.com/postman-docs/share-api-dashboard.jpg)
 
 ## Commenting on APIs
 
-You can comment on APIs in the Postman app or [web dashboard](https://app.getpostman.com).
+You can discuss your APIs with collaborators in the Postman app or [web dashboard](https://app.getpostman.com), keeping the conversation in context and available to other stakeholders. Anyone with access can comment, and comments will be visible in both the app and the dashboard.
 
-**Notes:**
+To leave a comment on an API using the Postman app, open __APIs__ in the left sidebar and select the API select the API you want to comment on. Click **Comments** at the top-right, enter your comment, and **Add comment** to publish it.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: oops thanks

File: src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md [hunk 3]
```diff
@@ -3,239 +3,254 @@ title: 'The API Workflow'
...
 
-This is a powerful feature that will let you jump-start your API workflow by creating collections from a schema. You can start by creating a schema or an API in Postman (sort of API-first development approach), and then start generating collections from the schema. It is so much easier for you to start working with your Schemas in Postman and use it for different workflows (like mocking, documentation, testing and monitoring).
+<img alt="Edit API" src="https://assets.postman.com/postman-docs/api-edit-options.jpg" width="250px"/>
 
-The **Generate Collections** button generates a collection which adheres to your schema and puts it in your workspace sidebar. Postman not only lets you generate a collection but it also enables you to add the collection as documentation, a test suite, and an integration or a contract test - all in one go. Once you click the **Generate Collections** button, the following screen appears:
+> You can also create APIs from your workspaces dashboard in the browser by navigating to __APIs__ and clicking __Create an API__.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md]: web browser, added detail

File: src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md [hunk 4]
```diff
@@ -3,239 +3,254 @@ title: 'The API Workflow'
...
 
-[![add existing mock](https://assets.postman.com/postman-docs/api-dev/Generate_Collection_Schema1.png)](https://assets.postman.com/postman-docs/api-dev/Generate_Collection_Schema1.png)
+You can also [version your APIs](/docs/postman/design-and-develop-apis/versioning-an-api/).
 
-People who have repetitive workflows can import their schemas in Postman and can generate collections on the run.
+## Defining an API
 
-### Importing a file
+You can define the structure of your API by creating a new schema or importing it from an existing file you can continue editing. You can also generate a collection from the schema.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/the-api-workflow.md]: linked

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md (hunk 1 above)
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: lol tbh I kept the edits here to a minimum as am also 🤯

**Comment by SueSmith** [comment]
@arlemi thanks for checking through this, think I've addressed everything..

File: src/pages/docs/postman/design-and-develop-apis/validating-elements-against-schema.md [hunk 5]
```diff
@@ -3,21 +3,40 @@ title: 'Validating Elements Against Schema'
...
+    url: "https://blog.postman.com/2019/05/17/postman-7-1-create-apis-directly-within-the-postman-app/"
+  - type: section
+    name: "Next Steps"
+  - type: link
+    name: "Viewing and analyzing APIs"
+    url: "/docs/postman/design-and-develop-apis/view-and-analyze-api-reports/"
 ---
 
-You can validate your API elements (documentation, tests, mock servers, or monitors) against the API schema. This helps keep your elements in sync with your API specification. If they do not match, you can see the list of issues that have been found in order to fix them.
+You can validate your API elements (documentation, tests, mock servers, monitors) against an API schema. This helps keep your elements in sync with your API specification. If they do not match, you can see the list of issues that have been found in order to fix them.
```
Comment (claireriley) [inline src/pages/docs/postman/design-and-develop-apis/validating-elements-against-schema.md]: Does the list of issues pop up in the app? maybe specify Postman will populate a list of issues that have been found

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 6]
```diff
@@ -1,141 +1,104 @@
...
-- [Removing an API from a workspace](#removing-an-api-from-a-workspace)
+- [Sharing APIs](#sharing-apis)
 - [Commenting on APIs](#commenting-on-apis)
+- [Viewing API events](#viewing-api-events)
 
-## Creating an API
+## Sharing APIs
 
-You can create an API in the workspace from the sidebar in the Postman app and from the workspaces dashboard. To create an API in the Postman app when no APIs exist in the current workspace, click **Create an API** in the sidebar.
+You can share an API via its workspace either in the Postman app or the [web dashboard](https://app.getpostman.com/dashboard). By sharing your workspace with collaborators, you can provide access to the API via designated roles with specific permissions, including Viewer and Editor.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: 👍 done

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 7]
```diff
@@ -1,141 +1,104 @@
...
 
-[![api create4](https://user-images.githubusercontent.com/5029719/71972999-bcae8280-3205-11ea-81f3-1482988b7426.png)](https://user-images.githubusercontent.com/5029719/71972999-bcae8280-3205-11ea-81f3-1482988b7426.png)
+> The Viewer role is only available on certain [Postman plans](https://www.postman.com/pricing).
 
-If you already have created APIs in your workspace, click '+' in the sidebar.
+If you share an API to a workspace, members of the workspace will have access to any shared elements linked to the API (for example documentation, test suites, monitors). Workspace members can view elements linked to an API but can only edit if an element is also shared. If you delete the API or remove it from the workspace, members will still have access to those shared elements unless you remove them also.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: Aye I totally agree, I believe "elements" is a recommendation from UX but we probably need to have a think about how to best describe this stuff for learning, maybe something Jacque could get into via collab squad.. Changed that second sentence.

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 8]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
 ---
 
-Business integration with the system and adding features to provide better user experience ends up creating more APIs. Their interactions become complex and traction of those becomes difficult. With the API ecosystem broadening, the need for detailed API analytics has arisen to facilitate providing feedback to *Design, Plan, Develop & Test* teams. API analytics help CXOs, Product managers, and API Architects to understand how their digital initiatives are performing.
+> __[API reports are available on Postman Business and Enterprise plans.](https://www.postman.com/pricing)__
 
-Postman's **Reporting** feature allows you to view data around the APIs created, collection executions, test failures and so on. Postman reports visualize your API data and provide insights to their use, their performance, troubleshooting, planning, and adherence to Service level agreements.
+You can access reports on your APIs in the Postman web dashboard. API reports visualize data on API activities including creation, collection execution, test runs, and more, providing insights on performance, troubleshooting, and SLA adherence, as well as team and organization metrics.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: done 👍

File: src/pages/docs/postman/design-and-develop-apis/managing-apis.md [hunk 9]
```diff
@@ -1,141 +1,104 @@
...
+<img alt="Edit API" src="https://assets.postman.com/postman-docs/api-edit-options.jpg" width="250px"/>
 
-> You can create an API only when you are signed in and online.
+Select a workspace and click __Share and Continue__.
 
-Alternatively, you can also create an API from your workspace in the [web dashboard](https://app.getpostman.com).
+![Share API to Workspace](https://assets.postman.com/postman-docs/share-api-to-workspace.jpg)
 
-[![api create3](https://user-images.githubusercontent.com/5029719/71973125-0008f100-3206-11ea-8643-a9904b236c51.png)](https://user-images.githubusercontent.com/5029719/71973125-0008f100-3206-11ea-8643-a9904b236c51.png)
+Configure the access levels for collaborators in your workspace and click __Save Roles__.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/managing-apis.md]: 👍

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 10]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
-* [Understanding API-level reports](#understanding-API-level-reports)
+## Team reports
 
-## Launching the reporting dashboard
+Team reports provide the following information:
 
-To launch the reporting dashboard, select the API from the APIs tab and click **Reports**. Reports of the selected API will be displayed. The following screen illustrates launching the reporting dashboard:
+* __Total number of APIs__ present in the team, including in private and public workspaces
+* __New APIs__ created within the team, not including new versions
+* __Active APIs__ having at least one request being sent through a linked collection
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: 😬 in all honesty I tried to keep this bit to a light redraft so am not sure on the detail

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 11]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
+* __Team size__ as the number of user accounts in the team
+* __Empty workspaces__ with no collections in them
+* __Active workspaces__ with at least one revision during a recent period, not including personal workspaces
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: again, am not at all sure

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 12]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
+[![team api req](https://assets.postman.com/postman-docs/Reports-TeamAPI.png)](https://assets.postman.com/postman-docs/Reports-TeamAPI.png)
 
-![reports main(https://assets.postman.com/postman-docs/API-Report-Team1.png)](https://assets.postman.com/postman-docs/API-Report-Team1.png)
+> In this example, 15 requests were sent on Jun 12. Over the next six days, no requests were sent. The 7-day average is 2.14 (15/7).
 
-In the above screen, select **Overview** to display organization-level metrics. It provides a statistical overview of the number of new and active APIs, team-size, and active and empty workspaces within a team. Click the corresponding section in the above screen for a detailed overview.
+## API reports
 
-## Understanding team-level reports
+API reports provide a graphical representation of data and enables you to filter the data by collection and duration (last 7 or 30 days). You can view reports for any individual collections you have linked ot an API.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: 😆 done

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 13]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
 
-Visibility into some of your most important team-level metrics helps you and your teams monitor the API usage over time and make necessary changes depending on the usage of APIs and addition and deletion of the APIs. Using the organization-level metrics dashboard, you can track changes in the APIs and take appropriate action.
+> Reports only track data from APIs you have shared to team workspaces. If your API is shared to a personal workspace you will not see reports for it.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: 👍

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 14]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
-### Active APIs
+[![team api req](https://assets.postman.com/postman-docs/Reports-AvgRespTime.png)](https://assets.postman.com/postman-docs/Reports-AvgRespTime.png)
 
-Displays the number of active APIs in the team. Postman considers an API active if at least one request was sent through a collection linked to an API.
+> In this example, the response time is shown for Jun 12. No response is received on other dates because no request is sent.
+>
+> Summary average response size may not match the average of the graph, because raw data is accounted for when calculating averages. In order to determine an accurate average, Postman does not calculate averages using data from already derived averages.
 
-### Team size
+Failed test runs are the number of failed tests for collection runs during a period of time. The summary average is calculated by taking into account all the days (even if no request is sent during a day).
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: much better 😬

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 15]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
+If you don't see any data in your reports, or your data is incomplete (for example you have empty response times or sizes) this may be due to one of the following reasons:
 
-* [Average response size](#average-response-size)
-* [Average response time](#average-response-time)
-* [Number of error responses](#number-of-error-responses)
-* [Traffic in Postman](#traffic-in-postman)
-* [Failed test runs](#failed-test-runs)
+* It's too early to check. The data refreshes every 24 hours.
+* You may not have linked a collection to your API.
+* You may not have sent a request—or you may have executed requests from monitoring, collection runs, or Newman, but not from the request builder in the Postman app.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: lol who knows 🙃

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 16]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
+* You may not have enabled __Save Responses__ in the __History__ tab in the Postman app.
+* App sync is not successfully communicating with the Postman servers.
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: aye there are so many possible issues and not sure on the ux

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md [hunk 17]
```diff
@@ -3,105 +3,119 @@ title: "Viewing and analyzing APIs"
...
 
-### Average response size
+> If none of the above reasons apply and you can't see data, please file a [github issue](https://github.com/postmanlabs/).
```
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: 👍

File: src/pages/docs/postman/design-and-develop-apis/validating-elements-against-schema.md (hunk 5 above)
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/validating-elements-against-schema.md]: clarified

**Comment by SueSmith** [comment]
@claireriley thanks for the detail! few things I need to check on to address them all but the ones I've marked resolved should be on the branch now

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: got clarification on this one and added

File: src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (SueSmith) [reply src/pages/docs/postman/design-and-develop-apis/view-and-analyze-api-reports.md]: got confirmation of this and yeah it is just the request builder at the moment

**Comment by SueSmith** [comment]
@claireriley clarified a couple of the outstanding bits but haven't had a response for some, think we're probably ok to publish then clarify later if we get more info..?

**Pull request merged**